ast/desugar/Desugar.cc

```diff
@@ -1367,14 +1367,14 @@ TreePtr node2TreeImpl(DesugarContext dctx, unique_ptr<parser::Node> what) {
             },
             [&](parser::Complex *complex) {
                 auto kernel = MK::Constant(loc, core::Symbols::Kernel());
-                core::NameRef complex_name = core::Names::Constants::Complex().data(dctx.ctx)->cnst.original;
+                core::NameRef complex_name = core::Names::Constants::Complex().cnst(dctx.ctx)->original;
```

Comment:
I anticipate wanting to update the docs to talk about why we have all these `NameRef` classes. (NameRef is a key data structure, so it's important to be documented in internals.md). If these are called `cnstData` or `dataCnst`, it will be easier to refer to them in the docs as "all these `.data*(...)` methods" and then they can figure out which are available. Also easier to grep for.

Can we update the name to make it more apparent that these helpers are more related to each other than they appear currently?

Reply:
> Can we update the name to make it more apparent that these helpers are more related to each other than they appear currently?

Sure, I can do that.

> I anticipate wanting to update the docs to talk about why we have all these NameRef classes.

? These classes already existed before.

Reply:
> These classes already existed before

Gotcha. I only just now noticed it, and found it confusing.

Feel free to not change in this PR, but I don't really know why we have `RawName` vs `UTF8` name kind and equate the two. I can add that to a list of changes to make in my free time if you don't want to do it.

Reply:
raw == utf8. I just pushed a change that unifies the terminology.